### DISCUSSION TITLE
apt.authelia.com returns HTTP 502

### DISCUSSION THREAD

**Discussion by scottrus** [authelia/authelia#8352] [General]
### Version

v4.38.17

### Deployment Method

Other

### Reverse Proxy

NGINX

### Description

Seeing 502 error from apt.authelia.com repository tonight

```root@auth:/etc/apt/sources.list.d# apt update
Hit:1 http://archive.ubuntu.com/ubuntu jammy InRelease
Hit:2 http://archive.ubuntu.com/ubuntu jammy-updates InRelease
Hit:3 http://archive.ubuntu.com/ubuntu jammy-security InRelease
Ign:4 https://apt.authelia.com/stable/debian/debian all InRelease
Ign:4 https://apt.authelia.com/stable/debian/debian all InRelease
Ign:4 https://apt.authelia.com/stable/debian/debian all InRelease
Err:4 https://apt.authelia.com/stable/debian/debian all InRelease
  502  Bad Gateway [IP: 104.21.15.110 443]
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
5 packages can be upgraded. Run 'apt list --upgradable' to see them.
W: Failed to fetch https://apt.authelia.com/stable/debian/debian/dists/all/InRelease  502  Bad Gateway [IP: 104.21.15.110 443]
W: Some index files failed to download. They have been ignored, or old ones used instead.
```

### Reproduction

Run `apt update` after following directions at https://apt.authelia.com/stable/debian/

### Build Information

```shell
Last Tag: v4.38.17
State: tagged clean
Branch: v4.38.17
Commit: 592ab519b4167dfe4462ee0a613d38d3e4b8efa0
Build Number: 35061
Build OS: linux
Build Arch: amd64
Build Compiler: gc
Build Date: Thu, 31 Oct 2024 09:37:31 +1100
Extra: 

Go:
    Version: go1.23.2
    Module Path: github.com/authelia/authelia/v4
    Executable Path: github.com/authelia/authelia/v4/cmd/authelia
```
```


### Logs (Authelia)


```shell
N/A
```

### Generative AI

No

### Pre-Submission Checklist

- [X] I agree to follow the [Code of Conduct](http://www.authelia.com/code-of-conduct)

- [X] This is a bug report and not a support request

- [X] I have read the security policy and this bug report is not a security issue or security related issue

- [X] I have either included the complete configuration file or I am sure it's unrelated to the configuration

- [X] I have either included the complete debug / trace logs or the output of the build-info command if the logs are not relevant

- [X] I have provided all of the required information in full with the only alteration being reasonable sanitization in accordance with the [Troubleshooting Sanitization](https://www.authelia.com/r/sanitize) reference guide

- [X] I have checked for related proxy or application logs and included them if available

- [X] I have checked for related issues and checked the documentation

**Comment by james-d-elliott** [comment]
This is hosted on infrastructure that we do not maintain, it's hosted by an external provider. They're probably just having an outage which will be resolved given time. https://www.getbalto.com
